modules/spring-config/src/main/java/org/mule/runtime/config/spring/dsl/model/ComponentModel.java

```diff
@@ -299,4 +321,25 @@ public int hashCode() {
     return result;
   }
 
+  //TODO WIP MULE-10252 useful for debugging, remove once completed.
+  public String toXml() {
```

Comment:
it should, but until I'm sure it will be merged I'll leave it there. Thanks for pointing it out.